joblib/test/test_parallel.py

```diff
@@ -477,7 +477,11 @@ def test_default_mp_context():
         if sys.platform == 'win32':
             assert_equal(p._mp_context.get_start_method(), 'spawn')
         else:
-            assert_equal(p._mp_context.get_start_method(), 'forkserver')
+            method = os.environ.get('JOBLIB_START_METHOD')
+            if method == 'forkserver':
+                assert_equal(p._mp_context.get_start_method(), 'forkserver')
+            else:
+                assert_equal(p._mp_context.get_start_method(), 'fork')
```

Comment:
We should check that `JOBLIB_START_METHOD` is always respected (even though we don't set it in travis and appveyor both for linux and windows.

If `JOBLIB_START_METHOD` is not set, the default should be `spawn` under win32 and `fork` otherwise.